sources/packages/api/src/database/entities/institution-location.model.ts

```diff
@@ -40,4 +40,10 @@ export class InstitutionLocation extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   institution: Institution;
+
+  @Column({
+    name: "institution_code",
+    nullable: false,
```

Comment:
Could you please remove the "nullable: false" or change it to nullable true, since it accepts null?